raiden/raiden_service.py

```diff
@@ -540,9 +541,67 @@ def connection_manager_for_token(self, token_address):
             raise InvalidAddress('token is not registered.')
         return manager
 
+    def leave_all_token_networks_async(self):
+        token_addresses = self.channelgraphs.keys()
+        leave_results = []
+        for token_address in token_addresses:
+            try:
+                connection_manager = self.connection_manager_for_token(token_address)
+            except InvalidAddress:
+                pass
+            leave_results.append(connection_manager.leave_async())
+        return leave_results
+
+    def close_and_settle(self):
```

Comment:
This could be greatly simplified if the logs are removed, do we need to introduce these messages here?